x/erc20/keeper/ibc_callbacks.go

```diff
@@ -88,12 +88,6 @@ func (k Keeper) OnRecvPacket(
 		data.Denom, data.Amount,
 	)
 
-	// If the coin denom starts with `factory/` then it is a token factory coin, and we should not convert it
-	// NOTE: Check https://docs.osmosis.zone/osmosis-core/modules/tokenfactory/ for more information
-	if strings.HasPrefix(data.Denom, "factory/") {
-		return ack
-	}
```

Comment:
nit: IMO this saves a token pair query and is more clear what we are filtering for.

Reply:
ok, i can re-add it